sys/arm64/arm64/locore.S

```diff
@@ -597,7 +618,12 @@ build_l1_block_pagetable:
 
 	/* Build the L1 block entry */
 	orr	x12, x7, #L1_BLOCK
+#ifdef ATTR_DEFAULT_LOW
+	orr	x12, x12, #(ATTR_DEFAULT_LOW)
+	orr	x12, x12, #(ATTR_DEFAULT_HIGH)
+#else
 	orr	x12, x12, #(ATTR_DEFAULT)
+#endif
```

Comment:
I still believe we should always be using ATTR_DEFAULT_LOW and ATTR_DEFAULT_HIGH here

Reply:
And I've already explained why I prefer doing it this way.

Reply:
It doesn't reduce the diff though; every assembly use of ATTR_DEFAULT still needs to be touched regardless, so why not do it the more-general way?

Reply:
There are 2 cases. I don't expect any more to be added as they are only needed to bootstrap the pagetables enough to get into C.

Reply:
So it's easy to fix now then.

Reply:
There is nothing to fix

Reply:
```suggestion
#if ATTR_DEFAULT_HIGH != 0
	orr	x12, x12, #(ATTR_DEFAULT_HIGH)
#endif
	orr	x12, x12, #(ATTR_DEFAULT_LOW)
```
is shorter, is clearer / more readable, and keeps more common code between Morello and plain kernels.

Reply:
I'm ok with the from @zxombie proposes unless it makes sense to upstream ATTR_DEFAULT_LOW and ATTR_DEFAULT_LOW in the short term.

Reply:
If going with that form I'd prefer the condition be `#if __has_feature(capabilities)`, since currently it's a bit weird that the HIGH use is guarded by there being a LOW.

Reply:
I can change it to `#if __has_feature(capabilities)`

Reply:
`ATTR_DEFAULT_LOW` and `ATTR_DEFAULT_HIGH` don't make sense upstream as `ATTR_DEFAULT` on non-morello arm64 fits within the operand of a singe `orr` instruction.